snow/engine/snowman/block/batched_vm.go

```diff
@@ -76,6 +79,11 @@ func GetAncestors(
 	for numFetched := 1; numFetched < maxBlocksNum && time.Since(startTime) < maxBlocksRetrivalTime; numFetched++ {
 		blk, err = vm.GetBlock(ctx, blk.Parent())
 		if err != nil {
```

Comment:
If we wanted to split this out - we could just break with `database.ErrNotFound` and log a `Warn` or `Error` for any other error